addons/mail/static/src/models/thread/thread.js

```diff
@@ -1126,6 +1174,22 @@ function factory(dependencies) {
             return [['unlink']];
         }
 
+        /**
+         * @private
+         * @returns {mail.message|undefined}
+         */
+        _computeLastTransientMessage() {
+            if (this.orderedMessages.length === 0) {
+                return [['unlink']];
+            }
+            for (let i = this.orderedMessages.length - 1; i >= 0; i--) {
+                const message = this.orderedMessages[i];
+                if (message.isTransient) {
+                    return [['link', message]];
+                }
+            }
```

Comment:
If none is found don't we also want to return unlink?

Reply:
Not sure a transient message can disappear but yes if it happens indeed we should unlink it